temp_workspace/input/sim_config.yaml

```diff
@@ -0,0 +1,37 @@
+  molecular_model:
+    voxel_size: 0.1         # [nm]
```

Comment:
are these string paths ever read by the TEM simulator? Or do we have freedom to change their names?
e.g. `voxel_size` to `pixel_size_ang`. If these are the "expected fields" to the TEM simulator, then I'm thinking we can have another layer above where standard params that are shared between different simulation routines are converted into "tem simulator" specific ones (units, etc).